lom/src/pluginmgr/pluginmgr_common/pluginmgr_helper.go

```diff
@@ -188,53 +220,62 @@ func (plmgr *PluginManager) run() error {
     for {
         select {
         case <-plmgr.systemStopChan: // system shutdown is received
-            lomcommon.LogInfo("RecvServerRequest() : Received system shutdoewn. Stopping plugin manager run loop")
+            LogInfo("RecvServerRequest() : Received system shutdown. Stopping plugin manager run loop")
             return nil
         case respObj := <-plmgr.responseChan: // response from plugin are received here and sent to engine via clientTx interface
             // If plugin manager is shutting down, do not send response to engine
             if plmgr.getShutdownStatus() {
-                lomcommon.LogInfo("In run(): Plugin manager is shutdown. Not sending response to engine %v", respObj)
+                LogInfo("In run(): Plugin manager is shutdown. Not sending response to engine %v", respObj)
```

Comment:
return nil after this statement ?

Reply:
I left it intentionally. After functional test, I will remove it based how much logs are generated.